notebooks/performance_monitoring.ipynb

```diff
@@ -0,0 +1,487 @@
+    "    process_postal_codes,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3",
+   "metadata": {},
+   "source": [
+    "### *Reminder: change the .env file to select the production database*"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4",
+   "metadata": {},
+   "source": [
+    "## Data exploration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5",
+   "metadata": {},
+   "source": [
+    "### Extract patients who are not called\n",
+    "Extract from ApiPrediction and ApiCallResponse. Select patients who had an prediction for a appointment, but were not called."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "session_object = init_session()\n",
+    "\n",
+    "with session_object() as session:\n",
+    "    predicted_patients = session.execute(\n",
+    "        select(\n",
+    "            ApiPrediction.start_time,\n",
+    "            ApiPrediction.prediction,\n",
+    "            ApiPrediction.id,\n",
+    "            ApiPrediction.patient_id,\n",
+    "            ApiPrediction.appointment_id,\n",
+    "            ApiCallResponse.call_status,\n",
+    "        )\n",
+    "        .outerjoin(ApiPrediction.callresponse_relation)\n",
+    "        .outerjoin(ApiPrediction.patient_relation)\n",
+    "    ).all()\n",
+    "\n",
+    "df = pd.DataFrame(\n",
+    "    predicted_patients,\n",
+    "    columns=[\n",
+    "        \"start_time\",\n",
+    "        \"prediction\",\n",
+    "        \"id\",\n",
+    "        \"patient_id\",\n",
+    "        \"appointment_id\",\n",
+    "        \"call_status\",\n",
+    "    ],\n",
+    ")\n",
+    "df.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7",
+   "metadata": {},
+   "source": [
+    "### Percentage and count of not-called patients"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df[\"start_time\"] = pd.to_datetime(df[\"start_time\"])\n",
+    "df[\"month\"] = df[\"start_time\"].dt.to_period(\"M\")\n",
+    "\n",
+    "current_month = pd.Timestamp(\"today\").to_period(\"M\")\n",
+    "df_months_filtered = df[df[\"month\"] < current_month]\n",
+    "\n",
+    "monthly_total = df_months_filtered[\"month\"].value_counts().sort_index()\n",
+    "\n",
+    "monthly_missing = (\n",
+    "    df_months_filtered[\n",
+    "        df_months_filtered[\"call_status\"].isna()\n",
+    "        | (df_months_filtered[\"call_status\"] == \"Niet gebeld\")\n",
+    "    ][\"month\"]\n",
+    "    .value_counts()\n",
+    "    .sort_index()\n",
+    ")\n",
+    "monthly_percent = (\n",
+    "    (monthly_missing / monthly_total * 100)\n",
+    "    .rename(\"percent_none_or_niet_gebeld\")\n",
+    "    .reset_index()\n",
+    ")\n",
+    "\n",
+    "months = df_months_filtered[\"month\"].unique()\n",
+    "months = months.astype(str)\n",
```

Comment:
Isn't this also just the value of the monthly_percent.index? Would be better to use the same dataframe for x and y, in case one of them is accidentally sorted differently